test/server_test.rb

```diff
@@ -27,6 +27,14 @@ def stop
   register 'mock', 'Rack::Handler::Mock'
 end
 
+class EventHookTest
+  def self.start
+  end
+
+  def self.stop
+  end
+end
```

Comment:
I think you can make this an anonymous class only available inside `ServerTest`: https://dev.to/factorial/a-trick-with-ruby-anonymous-classes-11pp

Reply:
Sure thing! I've turned it into an anonymous class.